.github/workflows/run-orch-smoketest.yml

```diff
@@ -30,6 +31,46 @@ jobs:
                                   -s secret/kdux/scp/staging/scp_service_account.json \
                                   -e test -v $DOCKER_IMAGE_TAG -n single-cell-portal-test \
                                   -c bin/run_orch_smoke_test.sh
+      - name: Notify Slack failure
+        id: notify-slack-on-fail
+        uses: slackapi/slack-github-action@v1.24.0
+        if: failure()
+        with:
+          channel-id: "C05JHJ0UA1M" #scp-alerts
+          payload: |
+            {
+              "blocks": [
+                { 
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "@channel :x: Orchestration smoke test: ${{ job.status }}\nhttps://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+                  }
+                }
+              ]
+            }
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACKBOT_TOKEN }}
+      - name: Notify Slack success
+        id: notify-slack-on-success
+        uses: slackapi/slack-github-action@v1.24.0
+        if: success()
+        with:
+          channel-id: "C05JHJ0UA1M" #scp-alerts
+          payload: |
+            {
+              "blocks": [
+                { 
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": ":green_check: Orchestration smoke test: ${{ job.status }}\nhttps://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+                  }
+                }
+              ]
+            }
```

Comment:
```suggestion
```

Non-blocking: notifying on success seems likely to cause [alarm fatigue](https://en.wikipedia.org/wiki/Alarm_fatigue).  Only alerting on failure seems best.

As is, this alert is significantly noisier than other alerts.  The other alerts in [#scp-alerts](https://broadinstitute.slack.com/archives/C05JHJ0UA1M), sourced from Mixpanel, only trigger on _positive_ cases.  That is, the alert is designed to detect an anomaly, and when it's detected (i.e. is positive), _only then_ send an alert.  It is very simple: to quickly notice a problem, people merely need to be members of #scp-alerts and not have the channel muted.  This matches the design of other DSP Slack alerting channels, e.g. [#workplace-resilience](https://broadinstitute.slack.com/archives/CA1481PRN).

This PR's design makes alerting more complex.  To ensure high signal-to-noise and thus avoid alarm fatigue, it requires each member of #scp-alerts to mute the channel.  It also requires all other alerts to include the `@channel` mention to ensure they get noticed by people.  The benefit is that it might mitigate false negatives by surfacing a canary -- i.e., if the 3-times-weekly negative alert doesn't post, maybe we're likelier to notice that #scp-alerts no longer includes these posts than we are to notice the no-longer-running job in GitHub Actions.  The cost is likelier alarm fatigue, per above.

I think cost exceeds benefit in this design, but I don't feel too strongly about it.  If we keep this design, let's at least update all Mixpanel alerts for #scp-alerts to include `@channel`.

Reply:
That's a great point, and one I thought about for a while.  I had it notify in all cases (and alert on failure) because I wanted a way to ensure that the job was running.  That being said, we can get the same information from the Actions tab in our repository - and GH Actions only notify on failure, or on success after repeated failures.  This PR as coded would change that behavior.  I think your concern is totally valid - I'll go ahead and remove this block.  Thanks for thinking this through!